pkg/target/kusttarget.go

```diff
@@ -25,12 +25,12 @@ import (
 
 	"github.com/ghodss/yaml"
 	"github.com/pkg/errors"
+	"sigs.k8s.io/kustomize/k8sdeps/transformer/hash"
+	"sigs.k8s.io/kustomize/k8sdeps/transformer/patch"
```

Comment:
This change will cause problem when vendoring `kustomize/pkg` in kubectl since they depend on `k8s.io/api-machinery`.

Reply:
that's right, still we can cut this down.  Will revisit later.